turbo.json

```diff
@@ -1,22 +1,23 @@
 {
   "$schema": "https://turbo.build/schema.json",
-  "ui": "tui",
+  "ui": "stream",
```

Comment:
I wonder reason

Reply:
Because it's more readable and not freezing on Windows and we can avoid case of missing some infos e.g. Warnings
But i can revert it here and open new PR with it

Reply:
Ok. Maybe it is addressed by the turborepo version update. Revert it now :)

Reply:
Ok, done.
Try to not forget about it.

I think you can merge latest packages after this PR, to `dev`